### DISCUSSION TITLE
PS Vita Live Area like launcher for Android [Suggestion]

### DISCUSSION THREAD

**Discussion by CubeVita** [EmiyaSyahriel/CrossLauncher#85] [Ideas]
Hello, I ask you to make a launcher that is similar to Live Area on PS Vita. You can do it when you want to(sorry if the category is wrong, etc.)

**Comment by EmiyaSyahriel** [comment]
PS Vita Live Area, huh? Looks interesting. It's just that I never owned PSV so I don't think I can copy the experience as near as it should even if I actually try making one, and in other hand I just burnt out working on CrossLauncher right now, even I just straight up didn't do the previous Xbox-like launcher request ( #21 ) once asked in the Issues page. Sorry to disappoint you but, I might not doing it, or if I am, might took a very long time that someone might already made a better one.

**Reply by xplshn** [reply]
>I am just burnt out working on CrossLauncher right now
@EmiyaSyahriel that's okay, You don't really need to implement what we people say, its YOUR project, don't let others take over it, it shouldn't divert from what you actually want it to be. I quite like it as it is.

**Reply by EmiyaSyahriel** [reply]
> It's YOUR project

@xplshn I mean I am not implementing this new idea in this exact project though, I am just stating that this idea is interesting but I am currently not in the capacity and mood doing it. That's it.

**Comment by Juoelenis** [comment]
yall lazy as fuck
